src/functionaltests/resources/features/subscriptionCRUD.feature

```diff
@@ -22,28 +22,31 @@ Feature: Test Subscription CRUD
   @RESTCreateSubscription
   Scenario: Create subscription with JSON object using REST API by POST method
     Given The REST API "/subscriptions" is up and running
-    When  I make a POST request with valid "JSON" to the  subscription REST API "/subscriptions"
+    When  I make a POST request with valid "JSON" to the subscription REST API "/subscriptions"
     Then  I get response code of 200
 
   @RESTReadSubscription
   Scenario: Read subscription using REST API by GET method
     Given The REST API "/subscriptions" is up and running
-    When  I make a GET request with subscription name "Subscription_Test" to the  subscription REST API "/subscriptions/"
+    When  I make a GET request with subscription name "Subscription_Test" to the subscription REST API "/subscriptions/"
     Then  I get response code of 200
     And   Subscription name is "Subscription_Test"
-    
+
   @RESTUpdateSubscription
-  Scenario: Update subscription using REST API by PUT method and validate updation
+  Scenario Outline: Update subscription using REST API by PUT method and validate updation
     Given The REST API "/subscriptions" is up and running
-    When I make a PUT request with modified notificationType as "MAIL" to REST API "/subscriptions"
+    When I make a PUT request with modified "<key>" as "<newValue>" to REST API "/subscriptions"
     Then I get response code of 200
-    And  I can validate modified notificationType "MAIL" with GET request at "/subscriptions/Subscription_Test"
-  
+    And  I can validate modified "<key>" "<newValue>" with GET request at "/subscriptions/Subscription_Test"
+
+# Values to be added to the variables given above
+  Examples:
+    | key              | newValue                         |
+    | notificationMeta | new.notification.meta/some/path/ |

```

Comment:
If only one set of example values are tested, then I don't see the point of having a scenario outline with examples. It's easier to read which parameters are sent in to the AND statement if it's written like before.

Reply:
I see it as it is easier to change what parameter to change when updating. The save "value" and "key" is used on multiple places thus, you need to update more values if you need to change them.

Reply:
I wish cucumber had better ways to add reused variables for the whole file.